### PR TITLE
Adding tree shaking guide.

### PR DESCRIPTION
**Pull request by malchata**
What's changed, or what was fixed?
- Added the tree shaking guide as requested by @addyosmani.

**Target Live Date:** 2018-06-14 (or later, there's no urgency on this)

- [x] This has been reviewed and approved by @addyosmani 
- [x] I have run `gulp test` locally and all tests pass.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele, @addyosmani

### CODE REVIEW

File: src/content/en/fundamentals/performance/optimizing-javascript/tree-shaking/index.md [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+{# wf_published_on: 2018-06-14 #}
+{# wf_blink_components: Blink>JavaScript #}
+
+# Reduce JavaScript Payloads with Tree Shaking {: .page-title }
+
+{% include "web/_shared/contributors/jeremywagner.html" %}
+
+Today's web applications can get pretty big, especially the JavaScript part of
+them. [As of
+mid-2018](https://httparchive.org/reports/state-of-javascript#bytesJs), HTTP
```
Comment (petele) [inline src/content/en/fundamentals/performance/optimizing-javascript/tree-shaking/index.md]: Nit: Put the link on the specific content it refers to. Right now you're linking to "mid-2018", but the content is really about the "median transfer size of JavaScript on mobile"

**Comment by addyosmani** [comment]
LGTM (we've been through tech review already) once we address Pete's nit.

**Comment by malchata** [comment]
Addressed it! Missed this feedback yesterday.

Should we circle in the team to give it a once over, quick?

**Comment by addyosmani** [comment]
After checking in with Sam he's happy with the state of the draft. Let's land this one and aim to deploy/publicize next Monday onwards.

cc @kosamari as an fyi this will be going up soonish :)

**Pull request merged**